package.json

```diff
@@ -7,7 +7,7 @@
   "license": "N/A",
   "private": true,
   "scripts": {
-    "dev": "cross-env TESTING_MODE=true GET_SUBMISSIONS_URL=/static/tmp.json next --inspect",
+    "dev": "cross-env TESTING_MODE=true GET_SUBMISSIONS_URL=http://localhost:7071/api/GetSubmissions SUBMIT_VOTE_URL=http://localhost:7071/api/SubmitVote next --inspect",
```

Comment:
This external dependency should be mentioned in the readme markdown to help other developers working on the site.